marc_to_solr/lib/augment_the_subject.rb

```diff
@@ -72,40 +95,70 @@ def indigenous_studies?(terms)
   # be assigned an Indigenous Studies term even though that entire term doesn't
   # appear in our terms list.
   def subfield_a_match?(term)
-    subfield_a = term.split(SEPARATOR).first
-    indigenous_studies_subfields["a"].include?(subfield_a)
+    subfield_a = normalize(term.split(SEPARATOR).first).gsub(/\.$/, '')
+    standalone_subfield_a_terms.include?(subfield_a)
   end
 
   ##
-  # For some subject terms, we need to match on all subfields
-  # E.g., Russia (Federation)-Civilization-Indian influences
-  def full_subject_term_match?(term)
-    indigenous_studies_terms.include? term.downcase.gsub(/[^\w\s#{SEPARATOR}]/, '')
+  # For some subfield terms, only a single subfield needs to match.
```

Comment:
Since the standalone subfield x's were in a separate, small worksheet on Google docs, we created the json version of this file manually. If this list becomes larger, we may need to revisit this process.